pints/_log_likelihoods.py

```diff
@@ -117,6 +117,120 @@ def __call__(self, x):
                       - np.sum(autocorr_error**2, axis=0) / (2 * sigma**2))
 
 
+class GaussianIntegratedUniformLogLikelihood(pints.ProblemLogLikelihood):
+    """
+    Calculates a log-likelihood assuming independent Gaussian-distributed noise
+    at each time point where it is assumed that the dependence on
+    :math:`\\sigma\\sim U(a,b)` has been integrated out of the joint posterior.
+
+    The likelihood is given in terms of the sum of squared errors:
+
+    .. math::
+        SSE = \\sum_{i=1}^n (f_i(\\theta) - y_i)^2
+
+    and is given up to a normalisation constant by:
+
+    .. math::
+        \\text{log } L = - n / 2 \\text{log}(\\pi) -
+            \\text{log}(2 (b - a) \\sqrt(2)) +
+            (1 / 2 - n / 2) \\text{log}(SSE) +
+            \\text{log}\\left[\\Gamma((n - 1) / 2, \\frac{SSE}{2 b^2}) -
+            \\Gamma((n - 1) / 2, \\frac{SSE}{2 a^2}) \\right]
+
+    where :math:`\\Gamma(u,v)` is the upper incomplete gamma function
+    as defined here: https://en.wikipedia.org/wiki/Incomplete_gamma_function
+
+    Arguments:
+
+    ``problem``
+        A :class:`SingleOutputProblem` or :class:`MultiOutputProblem`.
+    ``b``
```

Comment:
This does not match the signature `problem, lower, upper`

Reply:
Good spot! Have changed this.